bat/lib/mixerlib.bash

```diff
@@ -69,9 +69,9 @@ remove-package() {
 }
 
 download-rpm() {
-  mkdir -p ./rpms
-  pushd rpms
-  sudo curl -O $1
+  mkdir -p $BATS_TEST_DIRNAME/local-rpms
+  pushd $BATS_TEST_DIRNAME/local-rpms
```

Comment:
Again in the "I would write it this way", starting a subshell rather than using pushd/popd is reasonable.

```
download-rpm()(
    mkdir -p $BATS_TEST_DIRNAME/local-rpms
   cd $BATS_TEST_DIRNAME/local-rpms
   sudo -E curl -LO "$1"
)

As a general comment always having double quotes around things that are being expanded is good practice, it avoids splitting on white-space (technically on $IFS). Adding -E to sudo keeps http_proxy variables around.